custom/intrahealth/sqldata.py

```diff
@@ -5982,11 +5995,25 @@ def last_day_of_month(any_day):
 
         return max_date
 
+    def is_requested_location(self, location_name):
+        if self.config['location_id']:
+            location = SQLLocation.objects.filter(domain=self.config['domain'],
+                                                  location_id=self.config['location_id'])[0]
```

Comment:
this should be `SQLLocation.objects.get` - and what if it's not found?

Reply:
Thank you for comment. I've added changes in in next [pull request](https://github.com/dimagi/commcare-hq/pull/25595), 'filter' is changed to 'get' and exception related to lack of location is caught (if it occurs method returns False).

Reply:
thanks for the follow-up